### PR TITLE
Cool urls do not change

### PR DESCRIPTION
**Pull request by xMartin**
1. Removed the language URL prefix as we only have English for now. Could get tricky with redirecting. We'd actually want redirecting based on browser locale I'd say, which is not possible with all deployment options.
2. Added redirect configuration for Zeit.co hosting. Looks like we're lucky and the team I created there is still on the old pricing scheme which is free for teams if they need little performance 🙌

### CODE REVIEW

**Comment by gsambrotta** [comment]
Mhmm i don't know if i agree on move on in this way. 
We might not have translation right now but we would like to have them, rather soon since we have issues already open and @mamhoff saied he could work on it.
So i don't see so much the point of deploy this now in a solution which don't allow us to make translation later on. 
I would say make sense to find a solution now which will already work for the next step.

File: now.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "redirects": [
+    { "source": "/about\\.html", "destination": "/about" },
+    { "source": "/foundation/", "destination": "/about#foundation" },
```
Comment (Twissi) [inline now.json:4]: Does this also work for for urls without trailing slash?

File: now.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    { "source": "/sponsors\\.html", "destination": "/contribute#supporter" },
+    { "source": "/imprint\\.html", "destination": "/imprint" },
+    { "source": "/material\\.html", "destination": "https://learn.opentechschool.org/" },
+    { "source": "/berlin/", "destination": "/cities/berlin" },
+    { "source": "/leipzig/", "destination": "/cities/leipzig" },
+    { "source": "/zurich/", "destination": "/cities/zurich" },
+    { "source": "/brussels/", "destination": "/cities/brussels" },
+    { "source": "/bristol/", "destination": "/cities/bristol" },
+    { "source": "/romagna/", "destination": "/cities/romagna" },
+    { "source": "/handbooks/(.*)", "destination": "/guides" }
```
Comment (Twissi) [inline now.json:17]: Does this work for urls with multiple paths /*/*/*?

**Comment by Twissi** [comment]
I get what @gsambrotta is saying. If we remove translations out now they will probably never come back. Although we would be able to revert that change the effort to actually make it work will be to big that anyone will take care of it.

File: now.json (hunk 2 above)
Comment (xMartin) [reply now.json:17]: Yes.

File: now.json (hunk 1 above)
Comment (xMartin) [reply now.json:4]: No. But I think URLs without the slash should not exist, because it is added automatically.

**Comment by xMartin** [comment]
I'm afraid there more stuff to take care of until translated content und the i18n system fully works. Yes, it might get harder later on, because we might even have to change hosting from Zeit.co to somewhere else. More testing is needed, the switching UI and actual translations. Going live with a half-finished but in-place system doesn't make too much sense to me. We only know that it works if we make it work. We can wrap up the MVP now and launch and then take care of translations or do it now and wait with launching.

**Comment by Twissi** [comment]
Sorry, i think i missed the part, why does it not work with what we have right now?

**Comment by gsambrotta** [comment]
@Twissi we are not sure how much translations works because we never translate in other languages, not fully. So we cannot 100% the translation.

After a discussing with @xMartin, i figured out there will be more work and time to put into those translation system, since it will probably require some refactoring, the actual traslation and good testing.
We don't urgently need translation and we already agreed on going on live without.
So let's deploy without them for now and keep in account that when we want to introduce them again we might have also to consider new hosting solution. 🤞

**Pull request merged**